ui-frontend/packages/catalog-ui-search/package.json

```diff
@@ -56,7 +56,7 @@
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "4.0.0-alpha.56",
     "@material-ui/pickers": "3.2.10",
-    "@types/chai": "4.2.14",
```

Comment:
🤔 Did we already have the types and TS is still complaining about the import's explicit any?